wear/src/main/java/com/android/gpstest/wear/StatusScreen.kt

```diff
@@ -0,0 +1,136 @@
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.*
+import com.android.gpstest.Application
+import com.android.gpstest.library.data.FixState
+import com.android.gpstest.library.model.CoordinateType
+import com.android.gpstest.library.model.DilutionOfPrecision
+import com.android.gpstest.library.model.SatelliteMetadata
+import com.android.gpstest.library.model.SatelliteStatus
+import com.android.gpstest.library.ui.SignalInfoViewModel
+import com.android.gpstest.library.util.FormatUtils
+import com.android.gpstest.wear.theme.GpstestTheme
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+@OptIn(ExperimentalCoroutinesApi::class)
+@Composable
+fun StatusScreen(signalInfoViewModel: SignalInfoViewModel) {
+    val gnssStatuses: List<SatelliteStatus> by signalInfoViewModel.filteredGnssStatuses.observeAsState(
+        emptyList()
+    )
+    val location: Location by signalInfoViewModel.location.observeAsState(Location("invalid"))
+    val fixState: FixState by signalInfoViewModel.fixState.observeAsState(FixState.NotAcquired)
+    val satelliteMetadata: SatelliteMetadata by signalInfoViewModel.filteredSatelliteMetadata.observeAsState(
+        SatelliteMetadata()
+    )
+    val dop: DilutionOfPrecision by signalInfoViewModel.dop.observeAsState(
+        DilutionOfPrecision(
+            Double.NaN,
+            Double.NaN,
+            Double.NaN
+        )
+    )
+    GpstestTheme {
+        val listState = rememberScalingLazyListState()
+        Scaffold(
+            timeText = {
+                if (!listState.isScrollInProgress) {
+                    TimeText(
+                        timeSource = object : TimeSource {
+                            override val currentTime: String
+                                @Composable
+                                get() = if (location.time == 0L) "" else SimpleDateFormat("HH:mm:ss").format(
+                                    location.time
+                                )
+                        }
+                    )
+                }
+            },
+            positionIndicator = {
+                PositionIndicator(
+                    scalingLazyListState = listState
+                )
+            }
+        ) {
+            val contentModifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 8.dp)
+
+            ScalingLazyColumn(
+                modifier = contentModifier,
+                autoCentering = AutoCenteringParams(itemIndex = 0),
+                state = listState
+            ) {
+                item {
+                    CustomLinearProgressBar(fixState)
+                }
+
+                item {
+                    Text(
+                        text = String.format("Lat: ") + FormatUtils.formatLatOrLon(
+                            Application.app, location.latitude, CoordinateType.LATITUDE,
+                            Application.prefs
+                        )
+                    )

```

Comment:
Could you please move each of these `Text` into their own `@Composable` functions with a name for what it's showing?

It will simplify the code we see in the LazyColumn and make it easier to read the order of the elements.

So for example, this one would become:

```kotlin
@Composable
fun Latitude(location: Location) {
                    Text(
                        text = String.format("Lat: ") + FormatUtils.formatLatOrLon(
                            Application.app, location.latitude, CoordinateType.LATITUDE,
                            Application.prefs
                        )
                    )
}
```

...and here you would just have:

```kotlin
item {
    Latitude(location)
...
```

See the app StatusScreen for examples.